app/src/test/groovy/gov/hhs/cdc/trustedintermediary/etor/demographics/UnableToSendLabOrderExceptionTest.groovy

```diff
@@ -1,5 +1,6 @@
 package gov.hhs.cdc.trustedintermediary.etor.demographics
 
+import gov.hhs.cdc.trustedintermediary.etor.orders.UnableToSendLabOrderException
```

Comment:
This file needs to be moved to the `orders` package.

Reply:
Good catch. I'll move it